spec/20-http_request_header_format.md

```diff
@@ -352,6 +352,6 @@ Vendors receiving a `tracestate` request header MUST send it to outgoing request
 
 Following are allowed mutations:
 
-- **Update key value**. The value of any key can be updated. Modified keys MUST be moved to the beginning (left) of the list. This is the most common mutation resuming the trace.
-- **Add a new key/value pair**. The new key-value pair SHOULD be added to the beginning of the list.
+- **Add a new key/value pair**. The new key/value pair SHOULD be added to the beginning of the list.
+- **Update an existing value**. The value for any given key can be updated. Modified keys MUST be moved to the beginning (left) of the list. This is the most common mutation resuming the trace.
```

Comment:
This isn't really related to the PR but maybe while we're here, I don't understand what resuming the trace means here. Anyways since a spec probably can't predict what is more common or not, maybe can remove the sentence?

Reply:
I read this as participating in an existing trace by continuing it as opposed to starting off a new one.
I agree that this does not really fit into a technical specification but rather a usage guideline, for example, and would be fine with just removing that sentence.

Reply:
I think the sentence can just be removed as it is not a technical detail of the spec

Reply:
Removed in 6e72191.